nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/factory/Nd4j.java

```diff
@@ -3714,7 +3714,7 @@ public static INDArray create(byte[] data, long[] shape, long[]strides, char ord
     }
 
     /**
-     * See {@link #create(int[], long[], long[], char, DataType)}
+     * See {@link #, long[], long[], char, DataType)}
```

Comment:
Did you accidentally delete part of the name here?

Reply:
Indeed I did